scripts/k8s-storage/sc-rbd.yaml.in

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: k8s-storage-e2e-rbd-rwo
+provisioner: rook-ceph.rbd.csi.ceph.com
+parameters:
+   clusterID: @@CLUSTER_ID@@
+   pool: newrbdpool
```

Comment:
this should be replicapool as we create default pool as replicapool`

Reply:
yeah, manual tests were done with a standard Rook deployment